sources/platform/runtime/caching.md

```diff
@@ -6,9 +6,38 @@ page_description: How to speed up your builds using caching in Shippable
 
 # Caching
 
-You can optionally turn on caching for your CI (`runCI`) jobs to speed up your build by pulling your cache instead of installing all your dependencies. Please note that caching is currently not available for any other job type.
+You can optionally turn on caching for your jobs to speed up your build by pulling your cache instead of installing all your dependencies.
 
-## How caching works
+## Caching types
+Currecntly we support two types of cachings.
+
+* Node caching
+* Artifact caching 
+
+## Node caching
+
+### How node caching works
+
+You can enable node caching for any of your on-demand nodepool and then all the nodes created for that particular nodepool will be cached. This type of caching is not limited to only runCI jobs. This can be used for all the jobs running on that node.
```

Comment:
I think we need to explain this a little better. @manishas please review this and suggest improvements:

> When you use an on-demand node pool, Shippable dynamically provisions and terminates nodes for you based on your usage. The first build that runs on a freshly provisioned node can take longer to start running if it needs to pull in dependencies that are not already available on the node. When the node stays idle for a certain amount of time, Shippable will terminate it. Your dependencies will need to be pulled again when the next build runs on yet another freshly provisioned node.
>
> With node caching enabled, Shippable will pause and restart the same on-demand node for you instead of provisioning fresh machines. This allows your average build times to improve because dependencies can now be shared across a larger number of builds.
>
> Because this cache is tied to the node itself, both runCI and runSh jobs can take advantage of it.